test/unit/segments.cpp

```diff
@@ -512,6 +512,9 @@ class segments_test
     void
     run()
     {
+        url u;
```

Comment:
That's not in function with the tweaks because other tweaks are broken?

Reply:
This shouldn't have been here, it was temp code.